src/python/pants/backend/java/lint/google_java_format/rules.py

```diff
@@ -70,7 +81,7 @@ async def setup_google_java_format(
         Get(
             MaterializedClasspath,
             MaterializedClasspathRequest(
-                lockfiles=(tool.resolved_lockfile(),),
+                lockfiles=(lockfile,),
```

Comment:
Oh huh, this was violating the engine by using `open()` inside `lockfile_content`